charts/gitops-server/templates/admin-user-roles.yaml

```diff
@@ -36,9 +18,6 @@ rules:
   - apiGroups: ["helm.toolkit.fluxcd.io"]
     resources: [ "helmreleases" ]
     verbs: [ "get", "list", "watch", "patch" ]
-  - apiGroups: [ "notification.toolkit.fluxcd.io" ]
```

Comment:
Not blocking - can't validate what the page offers since its broken currently, but given you can suspend and resume alerts/providers it may make sense to have patch on these resources. We can always add back later.

Reply:
Nit: the notification resources are listed separately below in the ClusterRole, should likely be consistent

Reply:
Fixed

Reply:
@jpellizzari notification resources missing patch in the ClusterRole